Vagrantfile

```diff
@@ -114,11 +115,11 @@ ansible_provision = proc do |ansible|
     # Use monitor_address_block instead of monitor_interface:
     ansible.extra_vars.delete(:monitor_interface)
     ansible.extra_vars = ansible.extra_vars.merge({
-      cluster_network: "#{SUBNET}.0/16",
+      cluster_network: "#{CLUSTER_SUBNET}.0/16",
```

Comment:
I don't follow what you mean. For Linode, we aren't assigning any IP addresses. For background, my use-case with Linode is performance testing and I don't want the ceph daemons listening on any publicly accessible addresses. So all of the daemons should be listening on 192.168.0.0/16 (and ceph-ansible/vagrant cannot control what those addresses eventually are).